specs/commandline/verify.md

```diff
@@ -54,47 +68,68 @@ An example of `trustpolicy.json`:
 }
 ```
 
-In this example, only one policy is configured with the name `wabbit-networks-images`. With the value of property `registryScopes` set to `*`, this policy applies to all artifacts from any registry location. User can configure multiple trust policies for different scenarios. See [Trust Policy Schema and properties](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-policy) for details.
+For a Linux user, store file `trustpolicy.json` under directory `$HOME/.config/notation/`.
 
-### Verify signatures on a container image stored in a registry (Neither trust store nor trust policy is configured yet)
+For a Mac user, store file `trustpolicy.json` under directory `$HOME/Library/Application Support/notation/`.
 
-```shell
+For a Window user, store file `trustpolicy.json` under directory `C:\Users\<username>\AppData\Roaming\notation\`.
```

Comment:
```suggestion
For a Window user, store file `trustpolicy.json` under directory `%USERPROFILE%\AppData\Roaming\notation\`.
```

Reply:
Note: it is `$env:USERPROFILE` for PowerShell and `%USERPROFILE%` for cmd and file explorer.

Reply:
Note: Not all users have their user profiles in `C` drive. They can be in `D` drive or even in other folders.